pages/bestellung/ruecktritt.js

```diff
@@ -0,0 +1,126 @@
+
+Des Weiteren muss ein Rücktrittsgrund vorliegen. Das kann das fehlende Liefern der Ware oder einer beschädigten Ware sein – aber nur, wenn der Verkäufer seine Frist zur Nachbesserung/Neulieferung hat verstreichen lassen oder er eine Nacherfüllung vollständig verweigert.
+
+Ein  Rücktritt ist außerdem ausgeschlossen, wenn eine Teilleistung erfolgt ist und Du nicht nachweisen kannst, dass Du kein Interesse an der teilweisen Leistungserbringung hast.
+Auch ausgeschlossen ist, wenn die Ware nur in einem unerheblichen Maß beschädigt war. 
+Für weitere Infos dazu, klicke auf die jeweiligen Links weiter unten.
+`}
+        </p>
+      </StyledArticle>
+      <CopyToClipboard
+        text="Sehr geehrte Damen und Herren. Hiermit trete/n ich/wir von dem abgeschlossenen Vertrag über den Kauf folgender Waren/Einbringung
+     folgender Dienstleistungen:[] zurück. Bestellt am [] und erhalten am []. Grund dafür ist []. Die Voraussetzungen des Rücktritts liegen hier vor.
+     Bitte bestätigen Sie mir den Erhalt des Schreibens. 
+     [Name und Anschrift des Verbrauchers] [Datum] [Unterschrift]."
+        onCopy={() => alert("Anschreiben in die Zwischenablage kopiert!")}
+      >
+        <StyledKopierenButton>Anschreiben kopieren</StyledKopierenButton>
+      </CopyToClipboard>
+      <StyledLink href="/bestellung">
+        <StyledZurueckButton>zurück</StyledZurueckButton>
+      </StyledLink>
+    </StyledBody>
+  );
+}
+
+const StyledBody = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  gap: 20px;
+  background-color: #f3e8e8;
+  color: #572887;
+  border-radius: 10px;
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  width: 90vw;
+  height: 75vh;
+  margin-top: 1rem;
+  margin-bottom: 10px;
+  overflow: auto;
+`;
+
+const StyledBreadcrumb = styled.h3`
+  color: #572887;
+  font-size: 1rem;
+  font-weight: 400;
+  position: absolute;
+  top: 8rem;
+  left: 2rem;
+`;
+
+const StyledLink = styled(Link)`
+  text-decoration: none;
+`;
+
+const StyledIcon = styled(Icon)`
+  width: 40px;
+  height: 50px;
+  position: absolute;
+  top: 7.5rem;
+  right: 2rem;
+`;
+
+const StyledZurueckButton = styled.button`
+  color: #572887;
+  font-size: 1.2rem;
+  font-weight: 300;
+  font-style: normal;
+  text-decoration: none;
+  background-color: #e6ceee;
+  border-radius: 10px;
+  border: 1px solid rgba(163, 134, 182, 0.5);
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  height: 5vh;
+  width: 30vw;
+  cursor: pointer;
+  margin-bottom: 20px;
+`;
+
+const StyledKopierenButton = styled.button`
+  color: #572887;
+  font-size: 1.2rem;
+  font-weight: 300;
+  font-style: normal;
+  text-decoration: none;
+  background-color: #e6ceee;
+  border-radius: 10px;
+  border: 1px solid rgba(163, 134, 182, 0.5);
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  height: 15vh;
+  width: 40vw;
+  position: relative;
+  cursor: pointer;
+`;
+
```

Comment:
Please consider to create a more generic `StyledButton` component that can be used in different scenarios.
The buttons differ only in width and height, you can configure these as props.